pocs/utils/google/storage.py

```diff
@@ -315,8 +315,15 @@ def upload_observation_to_bucket(pan_id,
     Returns:
         str: A string path used to search for files.
     """
-    assert os.path.exists(dir_name)
-    assert re.match(r'PAN\d\d\d', pan_id) is not None
+    if os.path.exists(dir_name) is False:
+        raise OSError("Directory does not exist, cannot upload: {}".format(dir_name))
+
+    if re.match(r'PAN\d\d\d', pan_id) is None:
```

Comment:
This pattern will also match PAN0000000000. If you want to limit to 3 digits, you could use `r'PAN\d\d\d$'`.